src/Blog/BlogController.php

```diff
@@ -21,11 +21,6 @@ final class BlogController extends Controller
     private const POPULAR_TAGS_COUNT = 10;
     private const ARCHIVE_MONTHS_COUNT = 12;
 
-    protected function getId(): string
-    {
-        return 'blog';
```

Comment:
Wouldn't it result it `blog\blog`?

Reply:
No
![image](https://user-images.githubusercontent.com/6815714/85921891-30ae1700-b888-11ea-86b2-e90055fe3029.png)